pathexp/pathexp.go

```diff
@@ -283,27 +284,51 @@ func New(org, project string, envs, services, identities, instances []string) (*
 func Parse(raw string) (*PathExp, error) {
 	parts := strings.Split(raw, "/")
 
-	if len(parts) != 7 {
-		return nil, errors.New("wrong number of path segments")
+	if len(parts) > 7 {
+		return nil, errors.New("too many path segments")
 	}
-
 	if parts[0] != "" {
 		return nil, errors.New("path expressions must start with '/'")
 	}
+
 	// remove leading empty section
 	parts = parts[1:]
 
+	// identify double glob and replace remaining segments with glob
+	hasDoubleGlob := false
+	segments := make([]string, 6)
+	for i := range segments {
```

Comment:
two double splats should be invalid, as they'd be ambiguous:
`/jbowes/foo/**/default/**/foo` could be either:
- `/jbowes/foo/*/*/default/*/foo`
- `/jbowes/foo/*/default/*/*/foo`